contrib/monitoring/fdb_c_version.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# fdb_c_version.py
+#
+#  This source file is part of the FoundationDB open source project
+#
+#  Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argparse
+import ctypes
+import sys
+import platform
+import os
+
+def error(message):
+    print(message)
+    sys.exit(1)
+
+def get_version_string(library_path):
+    try:
+        lib = ctypes.cdll.LoadLibrary(os.path.abspath(library_path))
+    except Exception as e:
+        error('Could not load library %r: %s' % (library_path, e.message))
+
+    lib.fdb_get_error.restype = ctypes.c_char_p
+
+    try:
+        r = lib.fdb_select_api_version_impl(410, 410)
+        if r != 0:
+            error('Error setting API version: %s (%d)' % (lib.fdb_get_error(r), r))
+    except Exception as e:
+        error('Error calling fdb_select_api_version_impl: %s' % e.message)
+
+    try:
+        lib.fdb_get_client_version.restype = ctypes.c_char_p
+        version_str = lib.fdb_get_client_version()
+    except Exception as e:
+        error('Error getting version information from client library: %s' % e.message)
+
+    version_components = version_str.split(',')
```

Comment:
I got exception when using Python 3:
```
$ python ../fdb_c_version.py /usr/local/lib/libfdb_c.dylib 
Traceback (most recent call last):
  File "/Users/jzhou/src/foundationdb/../fdb_c_version.py", line 87, in <module>
    print(get_version_string(args.library_path))
  File "/Users/jzhou/src/foundationdb/../fdb_c_version.py", line 52, in get_version_string
    version_components = version_str.split(',')
TypeError: a bytes-like object is required, not 'str'
```

It seems this file is assuming Python 2.

Reply:
It is, this file is a bit old. I think we should target getting it committed and then we can migrate it to python 3.

Reply:
To be clear, I'm volunteering to do this work. It is easier to do if the file already exists and I can modify it.